tests/expectations/tests/objc_whitelist.rs

```diff
@@ -18,7 +17,7 @@ impl protocol_SomeProtocol for id {
     }
     unsafe fn protocolClassMethod() {
         msg_send!(
-            objc::runtime::Class::get("SomeProtocol").expect("Couldn't find SomeProtocol"),
+            objc::runtime::Class::get("SomeProtocol").expect("Couldn\'t find SomeProtocol"),
```

Comment:
Looks like there is too much escaping happening now.

Reply:
Nothing I can do about it unfortunately (unless I missed something)